src/main/java/calculator/domain/Calculator.java

```diff
@@ -0,0 +1,78 @@
+package calculator.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Calculator {
+    private Operands operands;
+    private Operators operators;
```

Comment:
```suggestion
    private final Operands operands;
    private final Operators operators;
```
필드를 불변하게 유지해주면 좀 더 안정적인 객체 사용을 유도할 수 있습니다 😄